ui/src/app/views/workspace-edit.tsx

```diff
@@ -390,8 +434,45 @@ export const WorkspaceEdit = fp.flow(withRouteConfigData(), withCurrentWorkspace
         cloneUserRole: false,
         loading: false,
         showUnderservedPopulationDetails: false,
-        showStigmatizationDetails: false
+        showStigmatizationDetails: false,
+        autocompleteLoading: false,
+        autocompleteDiseases: [],
+        diseaseHover: [],
+        dropDown: false,
+        searchTerm: '',
+        hovering: false
       };
+      this.searchTermChangedEvent = fp.debounce(300, this.diseaseSearch);
```

Comment:
this. searchTermChangedEvent should be defined as a class variable searchTermChangedEvent: Function;